sdk/quantum/quantum-jobs/tsconfig.json

```diff
@@ -2,7 +2,6 @@
   "extends": "../../../tsconfig.package",
   "compilerOptions": {
     "strict": true,
-    "target": "es5",
```

Comment:
this was needed because it depends on core-http. There is no need to bump the version I believe since it is still in beta.